client/lang/en.js

```diff
@@ -13,6 +13,7 @@ if (typeof(ss) === 'undefined' || typeof(ss.i18n) === 'undefined') {
     "LinkField.DELETE_ERROR": "Failed to delete link",
     "LinkField.ADD_LINK": "Add Link",
     "LinkField.ARCHIVE": "Archive",
-    "LinkField.DELETE": "Delete"
+    "LinkField.DELETE": "Delete",
+    "LinkField.SAVE_RECORD_FIRST": "Save record first"
```

Comment:
```suggestion
    "LinkField.SAVE_RECORD_FIRST": "Save the record before adding links"
```
This is a message for a content author rather than a developer, so it pays to be a little more verbose.
You'll need to update it everywhere else too.

Reply:
I'm on the fence between this and "Cannot add links until the record has been saved", so take your pick between those (or some variation of those)

Reply:
Updated